app/views/deregister_exemptions/new.html.erb

```diff
@@ -18,10 +18,11 @@
           <span class="error-message"><%= @deregister_exemptions_form.errors[:state_transition].join(", ") %></span>
           <% end %>
 
-          <%= f.label :state_transition, class: "form-label" do %>
+          <label class="form-label">
```

Comment:
W3C validation error was about the label containing a `for` to an id item that does not exist.